cmd/cape/cmd/token.go

```diff
@@ -52,14 +58,20 @@ func token(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func Token(functionID string, expires int) (string, error) {
+func Token(functionID string, expires int, owner bool) (string, error) {
 	privateKey, err := getOrGeneratePrivateKey()
 	if err != nil {
 		return "", err
 	}
 
+	var audience []string
+	if owner {
+		audience = append(audience, "owner")
+	}
+
 	token := jwt.NewWithClaims(jwt.SigningMethodRS256, jwt.RegisteredClaims{
 		Subject:   functionID,
+		Audience:  audience,
```

Comment:
Why do we use the audience field for this?

Reply:
The token "audience" in this case, is the "owner" themselves.

Reply:
I believe the audience claim is generally used to identify who the token is meant to be processed by? This seems like a token that has elevated privileges, so a custom claim might be in order?

Reply:
@bendecoste AFAIK the `aud` or audience is regarded as the `recipients` the token is intended for (a string array)

Reply:
Being said, given its use in oAuth I will add a custom claim

Reply:
> @bendecoste AFAIK the `aud` or audience is regarded as the `recipients` the token is intended for (a string array)

🤔 For example the token that auth0 issues to a user has the `aud` set as `something.capeprivacy.com` (login right now I think). Which is the server that processes the token, although the token is given to the user.